api/src/client/ApiFormationClient.test.ts

```diff
@@ -1223,6 +1223,514 @@ describe("ApiFormationClient", () => {
+                    State: members[1].addressState?.name,
+                    Zipcode: members[1].addressZipCode,
+                    Country: "US",
+                  },
+                },
+                {
+                  Name: members[2].name,
+                  Location: {
+                    Address1: members[2].addressLine1,
+                    Address2: members[2].addressLine2,
+                    City: members[2].addressCity,
+                    State: members[2].addressState?.name,
+                    Zipcode: members[2].addressZipCode,
+                    Country: "US",
+                  },
+                },
+              ],
+              Signers: [
+                {
+                  Name: incorporators[0].name,
+                  Title: "General Partner",
+                  Signed: true,
+                },
+              ],
+              ContactFirstName: formationFormData.contactFirstName,
+              ContactLastName: formationFormData.contactLastName,
+              ContactPhoneNumber: formationFormData.contactPhoneNumber,
+            },
+          },
+          { headers: { "Content-Type": "application/json" } }
+        );
+      });
+
+      it("posts to the endpoint with the api formation object when veteran-owned", async () => {
+        const stubResponse = generateApiResponse({});
+        mockAxios.post.mockResolvedValue({ data: stubResponse });
+        const incorporators = [
+          generateFormationIncorporator({
+            name: "faraz",
+            signature: true,
+            title: "General Partner",
+            ...generateFormationNJAddress({}),
+          }),
+        ];
+
+        const members = [
+          generateFormationMember({}),
+          generateFormationMember({}),
+          generateFormationMember({}),
+        ];
+        const formationFormData = generateFormationFormData(
+          {
+            agentNumberOrManual: "MANUAL_ENTRY",
+            hasNonprofitBoardMembers: true,
+            nonprofitBoardMemberQualificationsSpecified: "IN_BYLAWS",
+            nonprofitBoardMemberRightsSpecified: "IN_BYLAWS",
+            nonprofitTrusteesMethodSpecified: "IN_BYLAWS",
+            nonprofitAssetDistributionSpecified: "IN_BYLAWS",
+            isVeteranNonprofit: true,
+            members,
+            signers: undefined,
+            incorporators,
+          },
+          { legalStructureId: "nonprofit" }
+        );
+
+        const userData = generateUserDataForBusiness(
+          generateBusiness({
+            profileData: generateProfileData({
+              legalStructureId: "nonprofit",
+              businessPersona: "STARTING",
+            }),
+            formationData: generateFormationData({ formationFormData }),
+          })
+        );
+        const currentBusiness = getCurrentBusiness(userData);
+        await client.form(userData, "hostname.com/form-business");
+
+        expect(mockAxios.post).toHaveBeenCalledWith(
```

Comment:
There's a LOT going on in this test and its expectation that makes it hard to read and understand the cause-and-effect.

One easy way to make it easier to read would be to just expect the one thing we care about here:

`expect(getLastCalledWith(mockAxios.post)[1].Formation.BusinessInformation).toEqual("DomesticNonProfitVeteranCorporation")`

`getLastCalledWith` is currently a helper function on the frontend; you'd have to move it into shared test helpers instead to use it on the backend